src/config.ts

```diff
@@ -4,21 +4,34 @@ import argv from "yargs-parser";
 
 import packageJson from "../package.json" with { type: "json" };
 import fs from "fs";
+import { ReadConcernLevel, ReadPreferenceMode, W } from "mongodb";
 const { localDataPath, configPath } = getLocalDataPath();
 
 // If we decide to support non-string config options, we'll need to extend the mechanism for parsing
 // env variables.
-interface UserConfig extends Record<string, string | undefined> {
+interface UserConfig extends Record<string, unknown> {
```

Comment:
```suggestion
interface UserConfig {
```
nit: is there a benefit from extending Record?